packages/caip/src/adapters/banxa/index.ts

```diff
@@ -1,9 +1,7 @@
+import { toLower } from 'lodash'
```

Comment:
We should keep the original import - `import { namedExport } from 'lodah'` isn't properly tree-shaken

Reply:
caf718ac8e84aa0934f8f94ec3c8fbf210a0ab10